classes/help.php

```diff
@@ -155,32 +154,32 @@ function print_page() {
+													<td><?php _e( 'Enter the location the map should open to by default, when no location has been searched for. If you do not know the latitude and longitude of your starting location, enter the address in the provided text field and press "Geocode Address."', 'simplemap' ); // XSS ok. ?></td>
 												</tr>
 
 												<tr valign="top">
 													<td width="150">
-														<strong><?php _e( 'Auto-Load Database', 'SimpleMap' ); // XSS ok. ?></strong>
+														<strong><?php _e( 'Auto-Load Database', 'simplemap' ); // XSS ok. ?></strong>
 													</td>
 													<td>
-														<?php printf( __( '%s No auto-load:%s Locations will not load automatically.', 'SimpleMap' ), '<strong>', '</strong>' ); ?>
+														<?php printf( __( '%s No auto-load:%s Locations will not load automatically.', 'simplemap' ), '<strong>', '</strong>' ); ?>
 														<br/>
-														<?php printf( __( '%s Auto-load search results:%s The locations will load based on the default location, default search radius and zoom level you have set.', 'SimpleMap' ), '<strong>', '</strong>' ); ?>
+														<?php printf( __( '%s Auto-load search results:%s The locations will load based on the default location, default search radius and zoom level you have set.', 'simplemap' ), '<strong>', '</strong>' ); ?>
 														<br/>
-														<?php printf( __( '%s Auto-load all locations:%s All of the locations in your database will load at the default zoom level you have set, disregarding your default search radius. %s This option is not enabled if you have more than 100 locations in your database.%s', 'SimpleMap' ), '<strong>', '</strong>', '<em>', '</em>' ); ?>
+														<?php printf( __( '%s Auto-load all locations:%s All of the locations in your database will load at the default zoom level you have set, disregarding your default search radius. %s This option is not enabled if you have more than 100 locations in your database.%s', 'simplemap' ), '<strong>', '</strong>', '<em>', '</em>' ); ?>
 														<br/><br/>
 
-														<?php _e( 'If you leave the checkbox unchecked, then the auto-load feature will automatically move the map to the center of all the loaded locations. If you check the box, your default location will be respected regardless of the locations the map is loading.', 'SimpleMap' ); // XSS ok. ?>
+														<?php _e( 'If you leave the checkbox unchecked, then the auto-load feature will automatically move the map to the center of all the loaded locations. If you check the box, your default location will be respected regardless of the locations the map is loading.', 'simplemap' ); // XSS ok. ?>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 346 characters

<!-- Comment by [SideCI](https://sideci.com) -->